hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/ReplicationManager.java

```diff
@@ -664,11 +690,10 @@ private void updateMoveIfNeeded(final boolean isUnhealthy,
    * add a move action for a given container.
    *
    * @param cid Container to move
-   * @param srcDn datanode to move from
-   * @param targetDn datanode to move to
+   * @param mp MoveDataNodePair which contains source and target datanodes
    */
   public CompletableFuture<MoveResult> move(ContainerID cid,
```

Comment:
We should also fail move if ReplicationManager is not running or is not a leader?

Reply:
thanks  for pointing out this, i will add this check